libr/core/bin.c

```diff
@@ -394,20 +394,36 @@ static const char* get_compile_time(Sdb *binFileSdb) {
 	return timeDateStamp_string;
 }
 
+static int is_executable (RBinObject *obj) {
+	RListIter *it;
+	RBinSection* sec;
+	if (obj) {
+		if (obj->info && obj->info->arch)
```

Comment:
not sure if this case will be 100% true and maybe we want to check entrypoints too